chapters/iaas/aws/aws.md

```diff
@@ -11,7 +11,7 @@ Amazon in the following document:
 * <https://d0.awsstatic.com/whitepapers/aws-overview.pdf>
 
 We list the product in screenshots from their Product Web page panel
-in Figure next: @fig:aws-products-1, @fig:aws-products-2.
+in next Figure: @fig:aws-products-1, @fig:aws-products-2.
```

Comment:
As figures are specified with labels, we don't need the word "next" here.

Reply:
ok. I will delete all 'next' as long as the figure label is shown up